packages/keystone/fields/SymmetricEncryptedText/utils/decryptInfo.js

```diff
@@ -0,0 +1,28 @@
+const SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO = Symbol('SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO')
+
+
+/**
+ * @param item
+ * @param {string} fieldPath
+ * @param {CipherManagerResult} info
+ */
+function saveDecryptInfo (item, fieldPath, info) {
```

Comment:
This functions is so small. Why not to move it inside the field implementation file?

Reply:
outdated